src/app/models/gtypes/album_model.rs

```diff
@@ -45,9 +60,10 @@ mod imp {
 
     // Static array for defining the properties of the new type.
     lazy_static! {
-        static ref PROPERTIES: [glib::ParamSpec; 4] = [
+        static ref PROPERTIES: [glib::ParamSpec; 5] = [
             glib::ParamSpec::new_string("artist", "Artist", "", None, glib::ParamFlags::READWRITE),
             glib::ParamSpec::new_string("album", "Album", "", None, glib::ParamFlags::READWRITE),
+            glib::ParamSpec::new_uint("year", "Year", "", 0, 9999, 0, glib::ParamFlags::READWRITE),
```

Comment:
...hmm, okay I think I see why this is trickier than expected 😅 new_uint does not play nice with Options by the looks of it. if you want to keep that file as is I'm OK with that! the unwrap_or(0) below will do

Reply:
Yeah, [numbers cannot be optional](https://gtk-rs.org/gtk-rs-core/stable/latest/docs/glib/value/struct.Value.html), so I think it has to be unwrapped to something... I'm leaving it as is.

Reply:
right, fine by me :)